javaparser-testing/src/test/java/com/github/javaparser/ast/visitor/ModifierVisitorAdapterTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2016 The JavaParser Team.
+ *
+ * This file is part of JavaParser.
+ *
+ * JavaParser can be used either under the terms of
+ * a) the GNU Lesser General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ * b) the terms of the Apache License 
+ *
+ * You should have received a copy of both licenses in LICENCE.LGPL and
+ * LICENCE.APACHE. Please refer to those files for details.
+ *
+ * JavaParser is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ */
+
+package com.github.javaparser.ast.visitor;
+
+import org.jbehave.core.annotations.Then;
+
+import static org.junit.Assert.assertNotNull;
+
+public class ModifierVisitorAdapterTest {
+
+  @Then("ModifierVisitorAdapter requires no overrides")
```

Comment:
I do not understand what this test is doing. It instantiate an object and then it verifies it is not null?